include/aikido/planner/PostProcessorParams.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef AIKIDO_PLANNER_POSTPROCESSORPARAMS_HPP_
+#define AIKIDO_PLANNER_POSTPROCESSORPARAMS_HPP_
+
+#include <chrono>
+
+#include "aikido/planner/kunzretimer/KunzRetimer.hpp"
+#include "aikido/planner/parabolic/ParabolicSmoother.hpp"
+
+namespace aikido {
+namespace planner {
+
+/// Enum for which postprocessor is being used.
+enum class PostProcessorType
+{
+  HAUSER,
+  KUNZ
+};
+
+/// Trajectory postprocessing parameters. This struct can be used with the
+/// `ConcreteRobot` class, and let's the user specify the exact processor and
+//  and parameters to use.
+struct PostProcessorParams
```

Comment:
I'm honestly surprised that this compiles. The empty constructor `PostProcessorParams()` is ambiguous (does it map to the Kunz constructor or the Hauser constructor?).